netctl/netctl.go

```diff
@@ -844,8 +833,6 @@ func inspectBgp(ctx *cli.Context) {
 
 	hostname := ctx.Args()[0]
 
-	fmt.Printf("netctl. Inspecting bgp: %s\n", hostname)
```

Comment:
```
blah blah
{"foo":"bar"}
```

isn't JSON output that be parsed without manually tidying it up first.  If you request JSON with `-j`, the output should be parseable as JSON